frontend/src/components/grantaccess/GrantApplicationAdmin.vue

```diff
@@ -0,0 +1,182 @@
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+import { ErrorMessage, Field, Form as VeeForm } from 'vee-validate';
+import { object, string } from 'yup';
+import router from '@/router';
+import { AdminMgmtApiService } from '@/services/ApiServiceFactory';
+import { UserType, type FamApplication } from 'fam-app-acsctl-api';
+import type { FamAppAdminCreateRequest } from 'fam-admin-mgmt-api/model/fam-app-admin-create-request';
+import type { FamApplicationGetResponse } from 'fam-admin-mgmt-api/model/fam-application-get-response';
+import ApplicationSelect from '@/components/common/form/ApplicationSelect.vue';
+import Button from '@/components/common/Button.vue';
+import { IconSize } from '@/enum/IconEnum';
+import { Severity } from '@/enum/SeverityEnum';
+
+import { isLoading } from '@/store/LoadingState';
+import { setNotificationMsg } from '@/store/NotificationState';
+
+const defaultFormData = {
+    userId: '',
+    application: null as number | null,
+};
+const formData = ref(JSON.parse(JSON.stringify(defaultFormData))); // clone default input
+const formValidationSchema = object({
+    userId: string()
+        .required('User ID is required')
+        .min(2, 'User ID must be at least 2 characters')
+        .nullable(),
+    application: object().required('Application is required'),
+});
+
+const selectedApplication = ref<FamApplication>();
```

Comment:
I wonder if we could rename this variable maybe? cause it's the same name as the one in our application state store. Or I wonder if we could use `formData.application` directly, without adding an extra variable

Reply:
Thanks Catherine, the variable is not needed indeed. I'll remove it.